### DISCUSSION TITLE
clone Y axis to Z

### DISCUSSION THREAD

**Discussion by alestanga** [arkypita/LaserGRBL#1203] [Q&A] [answered]
I can't find an answer to this question:
is it possible to clone every command given to the Y axis to the Z axis?
it's a CNC machine in which both Y and Z are used as Y (two motors on the same axis, in parallell)
I've already made a C# program that does that, in the GCODE, but having the ability duplicate every command given to Y also to Z directly in LaserGRBL would enable me to use the command buttons to move both axis directly from the UI.
thanks.

**Comment by arkypita** [comment, marked as answer]
Usually this feature is implemented in hardware. Just wire enable step and direction pin of Y and Z together
